src/main/java/brave/opentracing/BraveActiveSpan.java

```diff
@@ -0,0 +1,199 @@
+            if (0 == refCount.decrementAndGet()) {
+                wrapped.finish();
+                scope.close();
+            }
+        }
+    }
+
+    @Override
+    public BraveActiveSpan.Continuation capture() {
+        return new BraveActiveSpan.Continuation();
+    }
+
+    @Override
+    public void close() {
+        deactivate();
+    }
+
+    public Span unwrap() {
+        return wrapped.unwrap();
+    }
+
+    @Override
+    public SpanContext context() {
+        return wrapped.context();
+    }
+
+    @Override
+    public BraveActiveSpan setTag(String key, String value) {
+        wrapped.setTag(key, value);
+        return this;
+    }
+
+    @Override
+    public BraveActiveSpan setTag(String key, boolean value) {
+        wrapped.setTag(key, value);
+        return this;
+    }
+
+    @Override
+    public BraveActiveSpan setTag(String key, Number value) {
+        wrapped.setTag(key, value);
+        return this;
+    }
+
+    @Override
+    public BraveActiveSpan log(Map<String, ?> fields) {
+        wrapped.log(fields);
+        return this;
+    }
+
+    @Override
+    public BraveActiveSpan log(long timestampMicroseconds, Map<String, ?> fields) {
+        wrapped.log(timestampMicroseconds, fields);
+        return this;
+    }
+
+    public static String toAnnotation(Map<String, ?> fields) {
+        return BraveSpan.toAnnotation(fields);
+    }
+
+    public static String joinOnEqualsSpace(Map<String, ?> fields) {
+        return BraveSpan.joinOnEqualsSpace(fields);
+    }
+
+    @Override
+    public BraveActiveSpan log(String event) {
+        wrapped.log(event);
+        return this;
+    }
+
+    @Override
+    public BraveActiveSpan log(long timestampMicroseconds, String event) {
+        wrapped.log(timestampMicroseconds, event);
+        return this;
+    }
+
+    @Override
+    public BraveActiveSpan log(String eventName, Object ignored) {
+        wrapped.log(eventName, ignored);
+        return this;
+    }
+
+    @Override
+    public BraveActiveSpan log(long timestampMicroseconds, String eventName, Object ignored) {
+        wrapped.log(timestampMicroseconds, eventName, ignored);
+        return this;
+    }
+
+    @Override
+    public BraveActiveSpan setBaggageItem(String key, String value) {
+        wrapped.setBaggageItem(key, value);
+        return this;
+    }
+
+    @Override
+    public String getBaggageItem(String key) {
+        return wrapped.getBaggageItem(key);
+    }
+
+    @Override
+    public BraveActiveSpan setOperationName(String operationName) {
+        wrapped.setOperationName(operationName);
+        return this;
+    }
+
+    public class Continuation implements ActiveSpan.Continuation {
+        Continuation() {
+            refCount.incrementAndGet();
+        }
+
+        @Override
+        public ActiveSpan activate() {
+            return new BraveActiveSpan(scope, wrapped, refCount, original);
+        }
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
until there's a use case for eq/hashCode maybe best to leave it out. ex you'll notice this is sensitive to refCount and not sure this is a valuable